### PR TITLE
code improvement based on checkstyle check

### PR DESCRIPTION
**Pull request by junchaowu**

### CODE REVIEW

**Comment by abhinav** [comment]
🆗

File: yarpc-core/src/main/java/io/yarpc/transport/http/HTTPRequestParser.java [hunk 1]
```diff
@@ -51,7 +51,8 @@
                     requiredProperty.getTtl(),
                     requiredProperty.getProcedure(),
                     requiredProperty.getService()),
-                    e);
+                    e
```
Comment (breerly) [inline yarpc-core/src/main/java/io/yarpc/transport/http/HTTPRequestParser.java]: trailing comma?

File: yarpc-core/src/main/resources/ExceptionMessages.properties [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
+FAIL_DECODE_REQUEST_HEADER=BadRequest: failed to decode "%s" request headers for procedure "%s" of service "%s" from caller "%s": %s
+FAIL_ENCODE_RESPONSE_HEADER=UnexpectedError: failed to encode "%s" request headers for procedure "%s" of service "%s from caller "%s"": %s
+FAIL_ENCODE_RESPONSE_BODY=UnexpectedError: failed to encode "%s" response body for procedure "%s" of service "%s" from caller "%s": %s
+FAIL_DECODE_RESPONSE_BODY=UnexpectedError: failed to decode "%s" response body for procedure "%s" of service "%s" from caller "%s": %s
+FAIL_DECODE_REQUEST_BODY=BadRequest: failed to decode "%s" request body for procedure "%s" of service "%s" from caller "%s": %s
+FAIL_ENCODE_REQUEST_BODY=failed to encode "%s" request body for procedure "%s" of service "%s": %s
+MISSING=BadRequest: missing %s
+NO_OUTBOUND=no configured outbound transport for service "%s"
+UNEXPECTED=UnexpectedError: error for procedure "%s" of service "%s": %s
+UNRECOGNIZED_HANDLER=BadRequest: unrecognized procedure "%s" for service "%s"
```
Comment (breerly) [inline yarpc-core/src/main/resources/ExceptionMessages.properties]: thx for sorting 👍

**Comment by breerly** [comment]
🚢 🇮🇹 

(were these changes made with a tool or by hand?)

**Comment by junchaowu** [comment]
make by IDE

File: yarpc-core/src/main/java/io/yarpc/transport/http/HTTPRequestParser.java (hunk 1 above)
Comment (abhinav) [inline yarpc-core/src/main/java/io/yarpc/transport/http/HTTPRequestParser.java]: I don't think Java allows trailing commas

Comment (breerly) [inline yarpc-core/src/main/java/io/yarpc/transport/http/HTTPRequestParser.java]: https://media4.giphy.com/media/Y2nbrJyAR6RiM/200_s.gif

**Pull request merged**